package/kedro_viz/server.py

```diff
@@ -70,35 +75,35 @@ def pretty_name(name):
     all_tags = set()
 
     for node in sorted(pipeline.nodes, key=lambda n: n.name):
-        task_id = "task/" + node.name.replace(" ", "")
+        task_id = _hash(str(node))
```

Comment:
If there is a node and a dataset with the same name, would `_hash` return the same hash ID for both of them? Is it worth adding the type to the hash seed, just to be sure that they'd get different IDs?